OpenRA.Game/Support/ColorUtils.cs

```diff
@@ -0,0 +1,176 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using OpenRA.Primitives;
+
+namespace OpenRA.Support
+{
+	public static class ColorUtils
+	{
+		public static Dictionary<Color, string> KnownColors = new()
```

Comment:
Wouldn't it be better to use `Enum.GetName<TEnum>(TEnum value)` and then "prettify it"?

Btw. you're missing Aqua and Fuchsia, is this intentional?

Reply:
> Btw. you're missing Aqua and Fuchsia, is this intentional?

They are duplicates (X11 vs. Windows naming) and would collide in a dictionary.

Reply:
Shouldn't these actually be defined in a fluent file?

Reply:
> Wouldn't it be better to use `Enum.GetName<TEnum>(TEnum value)` and then "prettify it"?

I can't get that to work. It seems you can only translate from enumeration numbers not values of the type inside.

Reply:
> Shouldn't these actually be defined in a fluent file?

Yes, that indeed makes more sense.